pkg/controllers/cloud/account_poller_test.go

```diff
@@ -0,0 +1,159 @@
+// Copyright 2022 Antrea Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cloud
+
+import (
+	"antrea.io/nephe/pkg/controllers/inventory"
```

Comment:
sort imports

Reply:
Done, messes up every time we make modifications to the file. Updated now.

Reply:
I still see imports not updated